src/content/blog/2025_fosdem.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Quantum Computing at FOSDEM 2025
+author: Purva Thakre
+day: 13
+month: 1
+year: 2025
+tags: 
+  - Community
+---
+
+**After five years, the Quantum Computing devroom returns to FOSDEM.**
+
+FOSDEM (Free and Open Source Software Developers' European Meeting) is a conference scheduled for 1-2 February 2025 at the ULB (Université libre de Bruxelles) Solboch Campus in Brussels, Belgium. It's a free event, and if you are in the area, you don't want to miss it! 
+
+The Quantum Computing devroom is slotted for the second half of Sunday 2 February 2025. We will be in room [K.4.401](https://fosdem.org/2025/schedule/room/k4401/).
```

Comment:
```suggestion
For the first time since 2020, FOSDEM will feature a Quantum Computing devroom, which is slotted for the second half of Sunday 2 February 2025. We will be in room [K.4.401](https://fosdem.org/2025/schedule/room/k4401/).
```